ELK_Listener/jjb_metrics_snippets.txt

```diff
@@ -0,0 +1,88 @@
+# This file shows what you can add to your jjb to
+# have it publish results out on the message bus.
+# This is not a complete JJB.  It shows what you
+# can add to be published in the build step. You
+# can decide to change publish whatever you would like.
+# This is currently set up to work with the ci listener
+# from this same directory.  Essentially, it shows how
+# to manually determine the variables of interest, then
+# publish them on the message bus using the Red Hat
+# CI jenkins plugin
+
+    builders:
+     - shell: |
+        #!/bin/bash
+
+        CREATE_TIME=$(date --utc +%FT%TZ)
+        .
+        .
+        <execution of job>
+        .
+        .
+        BKR_TESTS_EXEC=0
+        BKR_TESTS_FAILED=0
+        XUNIT_LINKS=''
+        # convert all recipeSets into Junit results (do not include failed installs)
+        for recipeSet in $(cat $WORKSPACE/recipeSets.txt); do
+            bkr job-results $recipeSet | xsltproc ./kernel/jenkins/helpers/recipeSet2junit.xml - > $WORKSPACE/${{recipeSet}}_xUnit.xml
```

Comment:
./kernel/jenkins/helpers comes from the internal tests/kernel repo.  The recipeSet2junit.xml file changes the output of the beaker job-results into a xUnit file.  The only purpose of this is to show an example of how I got the environment variables out of a job, in this case variables such as tests failed/executed.  If there is another way to get the variables of interest, this is by no means necessary.  For example, the messages I have seen from your jobs already have these variables properly set from what I can see.